src/kixi/heimdall/components/database.clj

```diff
@@ -82,6 +85,12 @@
       (if (coll? result)
         (map (partial into {}) reformatted)
         reformatted)))
+  (update! [this table what where]
+    (exec this (hayt/update table (hayt/set-columns
+                                   (reduce-kv #(assoc %1 (util/hyphen->underscore %2) %3)
```

Comment:
using that returns a vector (we tried first to do like other methods of protocol, doing apply map etc) and this function requires a hashmap as output
unless there's a way to use it we're missing?